Kernel/Devices/Storage/USB/SCSICodes.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2024, Leon Albrecht <leon.a@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Format.h>
+#include <AK/StringView.h>
+#include <AK/Types.h>
+
+namespace Kernel::SCSI {
+
+// SAM 4: 5.3.1 Status codes
+enum class StatusCode : u8 {
+    Good = 0x00,
+    CheckCondition = 0x02,
+    ConditionMet = 0x04,
+    Busy = 0x08,
+    // Obsolete = 0x10,
+    // Obsolete = 0x14,
+    ReservationConflict = 0x18,
+    // Obsolete = 0x22, was Command Terminated
+    TaskSetFull = 0x28,
+    ACAActive = 0x30,
+    TaskAborted = 0x40,
+};
+constexpr StringView to_string(StatusCode status)
+{
+    using enum StatusCode;
+    switch (status) {
+    case Good:
+        return "Good"sv;
+    case CheckCondition:
+        return "Check Condition"sv;
+    case ConditionMet:
+        return "Condition Met"sv;
+    case Busy:
+        return "Busy"sv;
+    case ReservationConflict:
+        return "Reservation Conflict"sv;
+    case TaskSetFull:
+        return "Task Set Full"sv;
+    case ACAActive:
+        return "ACA Active"sv;
+    case TaskAborted:
+        return "Task Aborted"sv;
+    default:
+        return "Unknown"sv;
```

Comment:
(nit: If you remove the `default:` and add a `VERIFY_NOT_REACHED()` below the switch, then clang's -Wswitch will warn if a new enum value is added but not added to the switch.)

Reply:
I would never trust SCSI HW to behave /hj

Reply:
Fair enough :)